test/web-platform-tests/to-upstream/cssom/getComputedStyle-hidden.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
Can you move this test to a new `html/rendering/the-css-user-agent-style-sheet-and-presentational-hints` folder (undernearth to-upstream)?